tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "sourceMap": true, /* Generates corresponding '.map' file. */
     "types": [
       "node",
+      "jest"
```

Comment:
This should be moved to the test/tsconfig.json file, otherwise it will expose wrong globals to the library